Readme update

MFlisar opened the pull request:
Updated the readme...

You need to make a release with the name "1.0.0", afterwards jitpack should work (but I'm not 100% sure here if I correctly defined the sub package names in the readme, would need to try this out afterwards)

vpratfr commented:
Hi

I have merged your 2 PRs and created the 1.0.0 release.

Can you check that the master branch is good?

Are you interested in getting commit access to this repo in order to directly make your changes here rather than your own repo and submit PRs?

MFlisar commented:
Needs small adjustments for jitpack (never sure what needs to be done for it, I'm doing this only ~3 times a year and often there are some new things to think of)

You can give my commit access, then I can fix this

vpratfr commented:
Ok. Done. You should be able to push. Feel free to append your credits in the readme if you wish.

MFlisar commented:
Finally, the jitpack build is working...

Added a batch to the readme,  updated the readme history, made a new release and now everything is working

vpratfr commented:
Cool. Thanks.
